packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -199,6 +214,18 @@ export interface CommonAutoScalingGroupProps {
    * @default none
    */
   readonly maxInstanceLifetime?: Duration;
+
+  /**
+   * Controls whether instances in this group are launched with detailed or basic monitoring.
+   *
+   * When detailed monitoring is enabled, Amazon CloudWatch generates metrics every minute and your account
+   * is charged a fee. When you disable detailed monitoring, CloudWatch generates metrics every 5 minutes.
+   *
+   * @see https://docs.aws.amazon.com/autoscaling/latest/userguide/as-instance-monitoring.html#enable-as-instance-metrics
+   *
+   * @default Monitoring.DETAILED
```

Comment:
```suggestion
   * @default - Monitoring.DETAILED
```
We use the `@default -` syntax to note that it's the default CloudFormation/AWS behavior, not CDK implementation.